File: src/targets/clojure/clj_http/fixtures/application-json.clj
(require '[clj-http.client :as client])

(client/post "http://mockbin.com/har" {:content-type :json
                                       :form-params {:number 1
                                                     :string "f\"oo"
                                                     :arr [1 2 3]
                                                     :nested {:a "b"}
                                                     :arr_mix [1 "a" {:arr_mix_nested {}}]
                                                     :boolean false}})

File: src/targets/csharp/restsharp/fixtures/application-json.cs
var client = new RestClient("http://mockbin.com/har");
var request = new RestRequest("", Method.Post);
request.AddHeader("content-type", "application/json");
request.AddParameter("application/json", "{\"number\":1,\"string\":\"f\\\"oo\",\"arr\":[1,2,3],\"nested\":{\"a\":\"b\"},\"arr_mix\":[1,\"a\",{\"arr_mix_nested\":{}}],\"boolean\":false}", ParameterType.RequestBody);
var response = client.Execute(request);